services/scheduler.go

```diff
@@ -172,7 +172,7 @@ func (ot *OneTime) RunJobAt(initr models.Initiator, job models.JobSpec) {
 		if err != nil {
 			logger.Error(err.Error())
 			initr.Ran = false
-			if err := ot.Store.Save(&initr); err != nil {
+			if err := ot.Store.UpdateInitiator(&initr); err != nil {
```

Comment:
Could we use `UPSERT` or does not fit into GORM's model?

Reply:
GORM doesn't appear to support a native upsert, I haven't looked into it any more than that. Could just be a `Exec` if desired.